.github/workflows/full-ci.yml

```diff
@@ -131,6 +131,12 @@ jobs:
           pip3 install pytest tarantool requests psutil pyyaml netifaces
 
       - name: Setup Docker
+        id: setupDocker
+        continue-on-error: true
+        uses: docker-practice/actions-setup-docker@master
+
+      - name: Retry setup Docker
```

Comment:
It looks ugly, and there's no comment on why it's needed.
Why only one extra try?

Reply:
Setup docker action fails often because it has 180 timeout for docker to bootstrap but it is sometimes not enough and it fails in the middle of an action. Second action skips previous steps and waits for another 180 seconds and continues setup. In the repo of an action there are issues about that problem but dev. doesn't seem to resolve them in near future.

Reply:
Will be fixed after switching to self-hosted runners